packages/cypress/tsup.config.ts

```diff
@@ -1,24 +1,45 @@
 import { defineConfig } from 'tsup';
 
-export default defineConfig((options) => ({
-  entry: [
-    'src/index.ts',
-    'src/support.ts',
-    'src/bin/archive-storybook.ts',
-    'src/bin/build-archive-storybook.ts',
-    'src/storybook-config/*.ts',
-  ],
-  splitting: false,
+const common = (options) => ({
   minify: !options.watch,
-  format: ['cjs', 'esm'],
+  splitting: false,
   dts: {
     resolve: true,
   },
   treeshake: true,
   sourcemap: true,
   clean: true,
-  platform: 'node',
   esbuildOptions(options) {
     options.conditions = ['module'];
   },
-}));
+});
+
+export default defineConfig((options) => [
+  // We want the cypress functions to be importable from both CJS or ESM files
```

Comment:
@skitterm I'm not sure if this is true TBH. I think it might be true of playwright, I cargo-culted it here.

Reply:
@tmeasday I'm not entirely up-to-speed on this, but the code here makes sure that we produce both cjs and esm versions of the index file. Is that correct?

If so, I think we want to keep both formats. In the Cypress docs, they show examples of both cjs and mjs files (for the `cypress.config.ts|js` file that imports things from this file).